portal/config/config.py

```diff
@@ -174,6 +175,16 @@ class BaseConfig(object):
     LOCKOUT_PERIOD_MINUTES = 30
     FAILED_LOGIN_ATTEMPTS_BEFORE_LOCKOUT = 5
 
+    RESTRICTED_FROM_PROMOTION_ROLES = [
```

Comment:
Will a list of strings in `application.cfg` work to configure this? (ie is a list of objects required?)